sys/net/application_layer/cord/Kconfig

```
@@ -0,0 +1,36 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_CORD
+    bool "Configure CoRE RD client"
+    depends on MODULE_CORD_COMMON || MODULE_CORD_EP || MODULE_CORD_EPSIM
```

As `cord_ep` and `cord_epsim` both depend on `cord_common`, I think it's enough just to check for the presence of this last one:
```suggestion
    depends on MODULE_CORD_COMMON
```